dcos/http.py

```diff
@@ -100,7 +100,8 @@ def _request(method,
                "please run: `dcos config set core.ssl_verify <value>`")
         description = config.get_property_description("core", "ssl_verify")
         if description is not None:
-            msg += "\n<value>: {}".format(description)
+            msg += "\n\ncore.ssl_verify: {}".format(description)
+        msg += "\n\npossible values: `True`, `False`, or a Path to CA"
```

Comment:
Lets lowercase 'Path' and remove one of the newlines. 
CA should be `path to a CA_BUNDLE file`